self_hosted.yml

```diff
@@ -2,7 +2,7 @@
 
 - hosts: self_hosted_first_host
   remote_user: root
-  gather_facts: no
+  gather_facts: yes
```

Comment:
We had `gather_facts: no` here because if we try to gather facts immediately and the host isn't up yet, the playbook will error out. If you look in the `wait_for_host_up` task, it waits for port 22 to come up on the host and then gathers facts.